specification/security/resource-manager/readme.md

```diff
@@ -106,6 +106,23 @@ tag: package-composite-v3
 
 The following packages may be composed from multiple api-versions.
 
+### Tag: package-preview-2025-05-04-preview
+
+These settings apply only when `--tag=package-preview-2025-05-04-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2025-05-04-preview'
+input-file:
+  - Microsoft.Security/preview/2025-05-04-preview/assessmentMetadata.json
+  - Microsoft.Security/preview/2025-05-04-preview/assessments.json
+suppressions:
+  - code: OperationsAPIImplementation
+    from: assessmentMetadata.json
+    reason: Suppression of OperationsAPI as it doesn't apply to this specific file.
+  - code: AvoidAdditionalProperties
+    from: assessments.json
+    reason: This is a property used across all API versions. changing it would be a breaking change, and is required for the SDK.
```

Comment:
>eason: This is a property used across all API versions. changing it would be a breaking change, and is required for the SDK. [](http://example.com/codeflow?start=5&length=124)

what property is? be specific

Reply:
also make your suppression specific with where clauses, to avoid allowing future violations